src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -1414,8 +1414,30 @@ private static TypeConverter GetIntrinsicTypeConverter(Type callingType)
                     }
                     else
                     {
+                        // Uri and CultureInfo are the only types that can be derived from for which we have intrinsic converters.
```

Comment:
We should probably add a comment to the dictionary definition as well, in case someone adds a new type in the future.